### PR TITLE
Use read only fs for Ouath and Job API component

### PR DESCRIPTION
**Pull request by Richard87**

### CODE REVIEW

File: pkg/apis/kube/config_maps.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"fmt"
 
+	_ "github.com/equinor/radix-operator/pkg/apis/test/initlogger"
```
Comment (nilsgstrabo) [inline pkg/apis/kube/config_maps.go:8]:
```suggestion
```
This is not a `test` file, so the logger should not be imported

File: pkg/apis/utils/resources/resources.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
+package resources
+
+import (
+	"github.com/rs/zerolog/log"
+	corev1 "k8s.io/api/core/v1"
+	resourcev1 "k8s.io/apimachinery/pkg/api/resource"
```
Comment (nilsgstrabo) [inline pkg/apis/utils/resources/resources.go:6]:
```suggestion
	"k8s.io/apimachinery/pkg/api/resource"
```
resource is not versioned

File: pkg/apis/utils/resources/resources.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		if resources.Requests == nil {
+			resources.Requests = corev1.ResourceList{}
+		}
+
+		resources.Limits[corev1.ResourceMemory] = mem
+		resources.Requests[corev1.ResourceMemory] = mem
+	}
+}
+
+func WithCPU(cpu string) ResourceOption {
```
Comment (nilsgstrabo) [inline pkg/apis/utils/resources/resources.go:31]: Consider using type resource.Quantity instead of string. Then you don't need to parse it

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Added a few suggestions. Also, the chart and app version should be updated.

File: pkg/apis/test/utils.go [hunk 4]
```diff
@@ -38,6 +46,23 @@ func NewTestUtils(client kubernetes.Interface, radixclient radixclient.Interface
...
+	logLevel, err := zerolog.ParseLevel(logLevelStr)
+	if err != nil {
+		logLevel = zerolog.InfoLevel
+	}
+
+	zerolog.ErrorStackMarshaler = pkgerrors.MarshalStack
+	zerolog.SetGlobalLevel(logLevel)
+	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.Kitchen})
+	zerolog.DefaultContextLogger = &log.Logger
+}
```
[the remark this answers is not in the document]
Comment (Richard87) [reply pkg/apis/test/utils.go:64]: fixed

File: pkg/apis/utils/resources/resources.go (hunk 3 above)
Comment (Richard87) [reply pkg/apis/utils/resources/resources.go]: fixed

File: pkg/apis/utils/resources/resources.go (hunk 2 above)
Comment (Richard87) [reply pkg/apis/utils/resources/resources.go]: fixed

**Pull request merged**